WcaOnRails/app/models/delegate_report.rb

```diff
@@ -24,7 +24,7 @@ def equipment_default
 Gen 2 Display: 0
 Gen 3 Display: 0"
   end
-
+  rubocop:disable LineLength
```

Comment:
Please move the enable/disable so it directly wraps the definition of the string (that means creating 2 pairs of enable/disable).

I could totally see someone adding a method inbetween these two methods and it would not be subject to our line length limit.